packages/frontend/src/components/userActious/gitcoin/YouDontHavePassport.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components'
+import { Button } from '../../buttons'
+import { Colors } from '@/styles/colors'
+import { BackButton } from '@/components/buttons/BackButton'
+import { Column, FormRow, FormWrapper, Row } from '@/components/form'
+import { urls } from '@/constants/urls'
+
+interface Props {
+  afterCreateClick: () => void
+}
+
+export const YouDontHavePassport = ({ afterCreateClick }: Props) => {
```

Comment:
Maybe we could rename it to something like: `MissingGitcoinPassport` or `UnavailableGitcoinPassport`, what do you think?